src/main/java/org/medicmobile/webapp/mobile/AppUrlVerifier.java

```diff
@@ -15,7 +15,7 @@
 import java.net.MalformedURLException;
 import java.util.concurrent.Callable;
 
-public class AppUrlVerifier implements Callable<AppUrlVerification> {
+public final class AppUrlVerifier implements Callable<AppUrlVerification> {
```

Comment:
Fixes a Spotbug failure. Some obscure security thing about throwing Exceptions from the constructors of non-final classes....